draft-packages/likert-scale-legacy/docs/LikertScaleLegacy.stories.tsx

```diff
@@ -26,70 +29,136 @@ export default {
 }
 
 const scale: Scale = [
+  {
+    value: -1,
+    label: "Not rated",
+  },
   {
     value: 1,
-    label: "Very poor",
+    label: "Strong Disagree",
   },
   {
     value: 2,
-    label: "Poor",
+    label: "Disagree",
   },
   {
     value: 3,
-    label: "Fair",
+    label: "Neither agree or disagree",
   },
   {
     value: 4,
-    label: "Good",
+    label: "Agree",
   },
   {
     value: 5,
-    label: "Very good",
+    label: "Strongly agree",
   },
 ]
 
-const labelId = "456"
-
-export const DefaultStory = () => {
+export const DefaultStory = args => {
   const [selectedItem, setSelectedItem] = useState<ScaleItem | null>(null)
+  const labelId = "456"
 
   return (
-    <div>
+    <>
       <div style={{ marginBottom: "40px" }} id={labelId}>
-        <Heading variant="heading-4">How would you rate this survey?</Heading>
+        <Heading variant="heading-4" color="dark">
+          How would you rate this survey?
+        </Heading>
       </div>
       <LikertScaleLegacy
         scale={scale}
         automationId="123"
         labelId={labelId} // Intended to match the id of the label
         selectedItem={selectedItem}
         onSelect={item => setSelectedItem(item)}
+        {...args}
       />
-    </div>
+    </>
   )
 }
-DefaultStory.storyName = "Default"
+DefaultStory.storyName = "Default (Kaizen Demo)"
 
-export const Reversed = () => {
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => {
   const [selectedItem, setSelectedItem] = useState<ScaleItem | null>(null)
-
   return (
-    <div>
-      <div style={{ marginBottom: "40px" }} id={labelId}>
-        <Heading variant="heading-4" color="white">
-          How would you rate this survey?
-        </Heading>
-      </div>
-      <LikertScaleLegacy
-        scale={scale}
-        automationId="123"
-        labelId={labelId} // Intended to match the id of the label
-        selectedItem={selectedItem}
-        onSelect={item => setSelectedItem(item)}
-        reversed
-      />
-    </div>
+    <Box mt={2}>
+      <StoryWrapper isReversed={isReversed}>
+        <StoryWrapper.Row rowTitle="Not rated">
+          <LikertScaleLegacy
+            scale={scale}
+            automationId="123"
+            labelId="1"
+            selectedItem={scale[0]}
+            onSelect={item => setSelectedItem(item)}
```

Comment:
I'll leave this decision to you, but as it stands the `onSelect` doesn't actually do anything for the example since we set the `selectedItem` to be constant.

We could either leave them constant and therefore remove the `onSelect` and state (probably okay since these are sticker sheets? Although we don't have a story with a proper code example; maybe we just change the default to have `docs: { source: { type: "code" } }`):
```suggestion
            onSelect={() => undefined)}
```

Or we have them all functional (which would likely mean different `useState`s for each scale.

Reply:
I decided to remove the onSelect and the state just because its just a sticker sheet.